src/System.Management.Automation/engine/InternalCommands.cs

```diff
@@ -2741,17 +2741,9 @@ public IEnumerable<CompletionResult> CompleteArgument(
             string wordToComplete,
             CommandAst commandAst,
             IDictionary fakeBoundParameters)
-        {
-            var strictModeVersionPattern = WildcardPattern.Get(wordToComplete + "*", WildcardOptions.IgnoreCase);
-
-            foreach (string version in s_strictModeVersions)
-            {
-                if (strictModeVersionPattern.IsMatch(version))
-                {
-                    yield return new CompletionResult(version);
-                }
-            }
-        }
+                => CompletionCompleters.EnumerateQuotedAndUnquotedCompletionText(
+                    wordToComplete,
+                    possibleCompletionValues: s_strictModeVersions);
```

Comment:
```suggestion
                    possibleCompletionValues: s_strictModeVersions,
                    resultType: CompletionResultType.ParameterValue);
```

Reply:
Good catch, fixed in b0a9dff8914cb2421f39a1366eea2eaae052fd78